### PR TITLE
Add info for Pure Storage CSI driver

### PR DESCRIPTION
**Pull request by sdodsley**
Add information for Pure Storage's CSI driver for Pure Service Orchestrator
```release-note
NONE
```
/kind documentation

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [OpenSDS](https://github.com/opensds/nbp/tree/master/csi) | `csi-opensdsplugin` | v1.0 | A Container Storage Interface (CSI) Driver for  [OpenSDS]((https://www.opensds.io/)) | Persistent | Read/Write Single Pod | Yes | Yes | Yes | ? | ?
 [Portworx](https://github.com/libopenstorage/openstorage/tree/master/csi) | `pxd.openstorage.org` | v0.3, v1.1 | A Container Storage Interface (CSI) Driver for [Portworx](https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/csi/) | Persistent | Read/Write Multiple Pods | Yes | No | Yes | Yes | ?
+[Pure Storage CSI](https://github.com/purestorage/helm-charts)| `pure-csi-plugin` | v1.0 | A Container Storage Interface (CSI) Driver for Pure Storage's [Pure Service Orchestrator](https://purestorage.com/containers) | Persistent | "Read/Write Single Pod" with FlashArray and "Read/Write Multiple Pods" with FlashBlade| Yes | No | No | ? | ?
```
Comment (msau42) [inline book/src/drivers.md]: Can you fill in the last 2 question marks

**Comment by sdodsley** [comment]
CLA signed

File: book/src/drivers.md (hunk 1 above)
Comment (sdodsley) [reply book/src/drivers.md]: Done

**Comment by msau42** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**